radix-engine/src/system/system_callback.rs

```diff
@@ -306,6 +306,8 @@ impl<C: SystemCallbackObject> KernelCallbackObject for SystemConfig<C> {
     where
         Y: KernelApi<Self>,
     {
+        let package_node_id = callee.package_address().as_node_id().clone();
+        update.copy_references.push(package_node_id);
```

Comment:
Can we move this to `// Additional global references` in `call_frame.rs`, so we can refactor them all together when re-enabling actor generics?